### PR TITLE
Pr fix windows installation

### PR DESCRIPTION
**Pull request by lyraik**
Fixed the executables extention to start the mavlink server on windows
@JonasVautherin

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -76,9 +76,14 @@ def platform_suffix(self):
     def mavsdk_server_filepath(self):
         """
         The location of the downloaded `mavsdk_server` binary
-        """
+        For windows this needs to be a .exe file
+        """   
+        if sys.platform.startswith('win'):
+            return 'mavsdk/bin/mavsdk_server.exe'
         return 'mavsdk/bin/mavsdk_server'
```
Comment (JonasVautherin) [inline setup.py:83]: Nitpick: I think this is slightly more readable:

```suggestion
        if sys.platform.startswith('win'):
            return 'mavsdk/bin/mavsdk_server.exe'
        else:
            return 'mavsdk/bin/mavsdk_server'
```

File: mavsdk/system.py [hunk 2]
```diff
@@ -352,21 +352,24 @@ def _start_mavsdk_server(system_address, port, sysid, compid):
...
+            if system_address:
+                bin_path_and_args.append(system_address)
+            p = subprocess.Popen(bin_path_and_args,
+                                    shell=False,
+                                    stdout=subprocess.PIPE,
+                                    stderr=subprocess.STDOUT)
+
+            logger = logging.getLogger(__name__)
+            log_thread = _LoggingThread(p.stdout, logger.debug)
+            log_thread.start()
```
Comment (JonasVautherin) [inline mavsdk/system.py:372]: hmm are you entirely sure that changing the indentation here does not change the behavior?

I'd rather keep the old indentation that I know works, if there is not a good reason to change it :blush:

**Review by JonasVautherin — CHANGES_REQUESTED** [review]
Thanks a lot for the contribution! Two small remarks, though :innocent:

File: setup.py (hunk 1 above)
Comment (lyraik) [reply setup.py:83]: I had It like your change before and it was suggested I change it to the other one 😆

**Review by JonasVautherin — APPROVED** [review]
Thanks a lot! :rocket:

**Comment by lyraik** [comment]
Thanks for adding the changes @JonasVautherin

**Pull request merged**